history/management/commands/pull_balance.py

```diff
@@ -39,12 +38,3 @@ def handle(self, *args, **options):
                                 deposited_amount_btc=deposited_amount_btc if ticker == 'BTC' else 0.00,
                                 deposited_amount_usd=deposited_amount_usd if ticker == 'BTC' else 0.00)
                     b.save()
-
-        for b in Balance.objects.filter(date_str='0'):
-            b.date_str = utc_to_mst_str(b.created_on)
-            b.save()
-
-        # normalize trade recommendations too.  merp
-        for tr in Trade.objects.filter(created_on_str=''):
-            tr.created_on_str = utc_to_mst_str(tr.created_on)
-            tr.save()
```

Comment:
Is it correct to remove this? Or do we need a way to distinguish some objects from the others?